File: response.py
class Response:
    def __init__(self, matcher, response_text, wildcard=False, reaction=False):
        self.matcher = matcher
        self.response_text = response_text
        self.wildcard = wildcard
        self.reaction = reaction

    def __str__(self):
        return "Response(matcher=\"{0}\", response_text=\"{1}\", wildcard={2}, reaction={3})".format(
            self.matcher, self.response_text, self.wildcard, self.reaction
        )

File: src/responses/response.ts
export class Response {
    readonly matcher: string
    readonly response_text: string
    readonly wildcard: boolean
    readonly reaction: boolean

    constructor(matcher: string, responseText: string, wildcard: boolean, reaction: boolean) {
        this.matcher = matcher
        this.response_text = responseText
        this.wildcard = wildcard
        this.reaction = reaction
    }
}
